controlpanel/frontend/jinja2/user-detail.html

```diff
@@ -153,23 +153,21 @@ <h2 class="govuk-heading-m govuk-error-summary-heading">
 </section>
 {% endif %}
 
-{% if request.user.has_perm('api.reset_mfa') %}
+{% if request.user.has_perm('api.add_superuser') %}
```

Comment:
Since you're already making changes in this area of the code, how about doing a couple of extra minor changes:
1. Display the `date_joined` field on the user detail page - would have been handy to have been able to see that here when we were looking into this support ticket
2. When setting someone as a superuser, mark them as `is_staff` too. Although looking at the view, this one isnt _quite_ as straightforward as I thought it would be, so could leave it as it is for now.

Reply:
Both have been added. Adding as staff at the same time isn't done in the best way but it's the only way I could get it to work.